AUTHORS

```diff
@@ -0,0 +1,7 @@
+Cédric Jeanneret <cjeanner@redhat.com>
+Haikel Guemar <hguemar@fedoraproject.org>
+Jason Guiditta <jason.guiditta@gmail.com>
+Jason Joyce <fuzzball81@gmail.com>
+Jason Joyce <jjoyce@redhat.com>
+Julie Pichon <jpichon@redhat.com>
+kyleecodes <kfieldsinmd@gmail.com>
```

Comment:
I believe this was automatically generated, we don't want to commit this.

Reply:
Correct it is, will remove it with updated patch